src/main/java/mate/academy/service/MovieSessionService.java

```diff
@@ -0,0 +1,14 @@
+package mate.academy.service;
+
+import java.time.LocalDate;
+import java.util.List;
+import mate.academy.model.MovieSession;
+
+public interface MovieSessionService {
+
+    MovieSession add(MovieSession movieSession);
+
+    MovieSession get(Long id);
```

Comment:
According to the checklist, the `get()` method in the service layer should return the actual entity, not an `Optional`. However, on the DAO layer, it should return `Optional`. Please ensure that the implementation of this method in the service layer directly returns the `MovieSession` entity, and the DAO layer returns an `Optional` that the service layer handles appropriately.